types/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "lib": ["es6"],
```

Comment:
I think it's es2015 in this case? But probably doesn't matter.

Reply:
changed. (i think it's [just an alias](https://www.typescriptlang.org/tsconfig/#high-level-libraries))